frontend/src/scenes/me/Settings/UserDetails.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { Input } from 'antd'
+import { useValues } from 'kea'
+import { userLogic } from 'scenes/userLogic'
+import { LemonButton } from 'lib/components/LemonButton'
+import { Field } from 'lib/forms/Field'
+import { VerticalForm } from 'lib/forms/VerticalForm'
+
+export function UserDetails(): JSX.Element {
+    const { userLoading, isUserDetailsSubmitting } = useValues(userLogic)
+
+    return (
+        <VerticalForm
+            logic={userLogic}
+            formKey="userDetails"
+            enableFormOnSubmit
+            style={{
+                maxWidth: 400,
+            }}
+        >
+            <Field name="first_name" label="Name">
+                <Input
```

Comment:
Let's switch to `LemonInput` while we're at it?

Reply:
I avoided that as it looks odd with the password fields stuck as antd inputs (they are used elsewhere and a bit more time intensive to change. Unless we don't mind it looking a bit odd?

Reply:
The trouble is that ant's inputs need to be wrapped in a function, as they pass the keyboard event to `onChange`, instead of just the changed value:

![2022-05-10 11 46 09](https://user-images.githubusercontent.com/53387/167600472-d41732e4-f538-43b2-b796-fd2d59b255b1.gif)

Reply:
I think it's fine if on this page, the password input looks a few pixels smaller.

Reply:
Just noticed that now as well 👍